docs/user-guide/src/bmo/automatic_secure_boot.md

```diff
@@ -0,0 +1,30 @@
+# Automatic secure boot
+
+The automatic secure boot feature allows enabling and disabling UEFI (Unified Extensible Firmware Interface) secure boot when provisioning a host. This feature requires supported hardware and compatible OS image. The current hardwares that support enabling UEFI secure boot are `iLO`, `iRMC` and `Redfish` drivers.
+
+Check also
+
+- [Ironic UEFI secure boot](https://docs.openstack.org/ironic/latest/admin/security.html#uefi-secure-boot-mode)
+- [Wikipedia UEFI secure boot](https://en.wikipedia.org/wiki/UEFI#SECURE-BOOT)
+
+## Advantages/Benefits
```

Comment:
Since it is answering "why" , we could rename it to:
```suggestion
## Why do we need a secure boot
```

Reply:
that was based on this review https://github.com/metal3-io/metal3-docs/pull/282#discussion_r982021125

Reply:
hmm, I am not sure really if advantages/benefits fits better than this..

Reply:
@kashifest what do you think ? not sure if we have a common pattern for other docs

Reply:
I am ok with this change

Reply:
Considering this has been reviewed by some folks, I took care of it addressing this and the below comment since @mboukhalfa is away and we are tight on populating as much content as possible for the user guide. Any left out comments/concerns can be addressed in a follow-up PR.